packages/onboarding/src/action-buttons/style.scss

```diff
@@ -65,11 +65,6 @@ button.action_buttons__button.components-button {
 		height: auto;
 	}
 
-	&.action-buttons__next {
-		color: var( --studio-white );
-		box-shadow: 0 0 0 1px var( --studio-blue-40 );
-	}
-
```

Comment:
Removed these styles because override other base styles. For instance, the `disabled` state:

<img src="https://user-images.githubusercontent.com/77539/143574851-f7e869fa-5f6f-4aa8-9187-5334d749f5d4.png" width="400px" />